client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -36,6 +38,21 @@ trait Client[F[_]] {
     */
   def fetch[A](req: F[Request[F]])(f: Response[F] => F[A]): F[A]
 
+  /** Submits a request, and provides a callback to process the response.
+    * Note that all errors, e.g. decoding a payload, client times out, etc.
+    *
+    * @param req An effect of the request to submit
```

Comment:
This isn't an effect.

```suggestion
    * @param req The request to submit
```

Reply:
Thanks, Ross, for correcting me.